examples/lookahead/README.md

```diff
@@ -5,3 +5,9 @@ Demonstration of lookahead decoding technique:
 https://lmsys.org/blog/2023-11-21-lookahead-decoding/
 
 More info: https://github.com/ggml-org/llama.cpp/pull/4207
+
+Sample command:
+
+```bash
+make -j && ./bin/llama-lookahead -hf ggml-org/Qwen2.5-Coder-3B-Q8_0-GGUF -p "// network server implemented in C\n// author: Peter Hacker\n\n#include" -e -ngl 99 -t 4 -n 512 -c 4096 -kvu
```

Comment:
Should this perhaps be using cmake instead as unless users have set `LLAMA_MAKEFILE=1` they would get the deprecation warning if they use make right?

Reply:
I just copied the command that I use inside my CMake build dir - not intended to be used with the deprecated `Makefile`.

Updated the command to not involve the build step - should be less confusing now.